reana_cluster/backends/kubernetes/templates/persistentvolumeclaims/cvmfs-alice.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: csi-cvmfs-atlas-pvc
+spec:
+  accessModes:
+  - ReadOnlyMany
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
The `1Gi` value doesn't affect the storage, but is required according to the clouddocs instructions